src/util/types.ts

```diff
@@ -850,6 +850,19 @@ export interface BorderOptionMixin {
     borderMiterLimit?: number
 }
 
+export type ColorBy = 'seriesName' | 'seriesIndex' | 'seriesId' | 'name' | 'dataIndex' | 'id';
+
+// Sunburst color mapping on levels and items are yet to be implemented
+// export type SunburstColorBy = ColorBy | 'inherit' | 'childIndex' | 'lighter';
```

Comment:
To-be-implemented feature can stay in its own feature branch. Commented code is dead code and is a noise. Git will help you merge changes from dev branch.